corehq/form_processor/models/cases.py

```diff
@@ -342,6 +342,13 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def natural_key(self):
+        """
+        Django recommends always returning a tuple in natural_key methods:
```

Comment:
I know we read the same sentence together on a call, but since there isn't an enforcement of this at the code level, I was intentionally shying away from saying "requires", but I'm open to being convinced why "recommends always returning" isn't sufficient 😁

Reply:
The docs say

> That method should always return a natural key tuple...

To me that sounds more like a requirement than a recommendation. The fact that we have gotten away with returning something other than a tuple is somewhere between convenient coincidence and wildly lucky.

Reply:
52a70116718b0a05dd244e1e255419dfc0f6eb77